app/renderer/components/tabs/content/audioTabIcon.js

```diff
@@ -66,19 +101,14 @@ class AudioTabIcon extends React.Component {
       className={css(styles.audioTab__icon)}
       symbol={this.audioIcon}
       onClick={this.toggleMute}
+      ref={this.setRef}
```

Comment:
Can we use `ref={(node) => { this. element = node }}`?

I don't see a need to use findDOMNode for this scenario.

Reply:
@NejcZdovc I'm pretty sure in this case `node` would be the instance of the component class and not the required underlying DOM element, since that `ref={}` attribute is on a custom React component, not a normal HTML element. `findDOMNode` traverses the rendering structure to get the ultimate html element

Reply:
tbh all this could be refactored to a simpler structure, but that would be a subsequent PR